docker-compose-configuration/metaphactory-docker-compose/rds-local/docker-compose.base.yml

```diff
@@ -0,0 +1,58 @@
+version: "2.2"
+services:
+  metaphactory:
+    ports:
+      - "10214:8080"
+    depends_on:
+      - blazegraph
+    container_name: "${COMPOSE_PROJECT_NAME}-metaphactory"
+    image: "${METAPHACTORY_IMAGE}"
+    restart: unless-stopped
+    mem_limit: 1g
+    mem_reservation: 1g
+    environment:
+      - PLATFORM_OPTS=-Dconfig.environment.sparqlEndpoint=http://${COMPOSE_PROJECT_NAME}-blazegraph:8080/blazegraph/sparql ${METAPHACTORY_OPTS}
+      - JAVA_OPTS=
+      - GENERATE_METAPHACTORY_PASSWORD=${GENERATE_METAPHACTORY_PASSWORD}
+    networks:
+      - default
+      - metaphactory_network
+    volumes:
+      - metaphactory-apps:/apps
+      - ${RDS_SCRIPT_ROOT_DIRECTORY}/rds-local-runtime:/runtime-data:rw
+      # Applications
+      # - ${RDS_SCRIPT_ROOT_DIRECTORY}/rds-local-apps/rds-demo:/apps/rds-demo
```

Comment:
these lines should go into a `docker-compose.override.yml` file and be an extension to the basic platform setup

Reply:
done.